### PR TITLE
[Homework #5] - CSS-Advanced by Usmonbek Rustamov

### PR DESCRIPTION
**Pull request by usmonbek-umarbekovich**
Hello,

I have implemented the website. It is responsive and has some nice animations.
Let me know what you think, thank you

### CODE REVIEW

File: homeworks/usmonbek-rustamov/05-css-advanced/index.html [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    <script type="module" src="script.js"></script>
+    <title>Victor Alvarado | Software Engineer</title>
+  </head>
+  <body class="light-theme">
+    <!-- Header -->
+    <header id="main-header" class="header">
+      <div class="header__container container flex flex_direction_v">
+        <a class="theme-toggler btn btn_shape_round" id="theme-toggler">
+          <svg class="theme-toggler__icon">
+            <use href="./assets/icons/sun.svg#sun"></use>
```
Comment (VitalikMelnyk) [inline homeworks/usmonbek-rustamov/05-css-advanced/index.html:16]: Please, fix icons.
Screenshot 2023-07-19 at 16 05 04
Screenshot 2023-07-19 at 16 05 00

**Review by VitalikMelnyk — CHANGES_REQUESTED** [review]
I have checked your PR.
Just need to fix one issue with icons.

File: homeworks/usmonbek-rustamov/05-css-advanced/index.html (hunk 1 above)
Comment (usmonbek-umarbekovich) [reply homeworks/usmonbek-rustamov/05-css-advanced/index.html:16]: I checked it on many browsers and the icons are showing.  Can you tell me which browser did you use?

**Comment by usmonbek-umarbekovich** [comment]
I think that should fix the issue if you're using older version of the browser. Please let me know if it works for you now

File: homeworks/usmonbek-rustamov/05-css-advanced/index.html (hunk 1 above)
Comment (VitalikMelnyk) [reply homeworks/usmonbek-rustamov/05-css-advanced/index.html]: Not working in Safari and Chrome.

**Comment by VitalikMelnyk** [comment]
@usmonbek-umarbekovich, the same issue with icons.
I have the latest version of Chrome.
Maybe, replace a logic with the `use` tag to something else because the same problem occurred.

**Comment by usmonbek-umarbekovich** [comment]
I changed use to img. Now it should work

**Review by VitalikMelnyk — APPROVED** [review]

**Comment by VitalikMelnyk** [comment]
Well Done @usmonbek-umarbekovich!
Approved.

**Pull request merged**